### PR TITLE
[EMCAL-760] Stand-alone publisher for digits

### PR DESCRIPTION
**Pull request by mfasDa**
Trivial stand-alone publisher for digits, based on the
publisher in the RecoWorkflow. Needed for the QC
in simulation.

### CODE REVIEW

**Review by shahor02 — COMMENTED** [review]
@mfasDa I would suggest to add the HBFUtilsInitializer lines which allows to provide consistent firstTForbit and tf creation time both in the MC and real data digits injection. Just grep for this word, you will see the line to add.

**Comment by mfasDa** [comment]
Hi @shahor02, thanks for pointing out! I took it from the EMCAL reco workflow - it should be done now (also for the cell reader workflow.

File: Detectors/EMCAL/workflow/src/digit-reader-workflow.cxx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+#include "CommonUtils/ConfigurableParam.h"
+
+using namespace o2::framework;
+using namespace o2::emcal;
+
+// we need to add workflow options before including Framework/runDataProcessing
+void customize(std::vector<ConfigParamSpec>& workflowOptions)
+{
+  std::vector<ConfigParamSpec> options{{"disable-mc", VariantType::Bool, false, {"Do not propagate MC labels"}},
+                                       {"configKeyValues", VariantType::String, "", {"Semicolon separated key=value strings"}}};
```
Comment (shahor02) [inline Detectors/EMCAL/workflow/src/digit-reader-workflow.cxx:28]: You missed here `o2::raw::HBFUtilsInitializer::addConfigOption(options);`

Comment (mfasDa) [reply Detectors/EMCAL/workflow/src/digit-reader-workflow.cxx:28]: Done, thanks for spotting!

**Comment by mfasDa** [comment]
@shahor02 Do you think it can be merged now?

**Review by shahor02 — APPROVED** [review]

**Pull request merged**